packages/events/defaultSubscribers/dashboard.js

```diff
@@ -39,6 +39,17 @@ module.exports = {
         }
       }
     ],
+    "compile:succeed": [
+      async function ({ result }) {
+        try {
+          const publishLifecycle = await this.messageBusClient.publish({
+            type: "workflow-compile-result",
+            payload: result
+          });
+          publishLifecycle.abandon();
+        } catch (err) {}
+      }
+    ],
```

Comment:
Yeah, given that this is being sent from the `events` package, I think it would be wise to change this out for a more generic event envelope.

Reply:
Nice catch on the `.abandon()`, too.

Reply:
Yup! It's now `CliEventMessage`.